pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -1697,6 +1697,9 @@ func (fs *eosfs) convert(ctx context.Context, eosFileInfo *eosclient.FileInfo) (
 				},
 			},
 		},
+		ArbitraryMetadata: &provider.ArbitraryMetadata{
+			Metadata: eosFileInfo.Attrs,
+		},
```

Comment:
@gmgigi96 are `sys.` attributes discarded in this field ?

Reply:
now yes ;)